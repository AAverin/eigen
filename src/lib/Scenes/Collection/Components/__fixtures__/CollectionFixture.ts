export const CollectionFixture = {
  slug: "street-art-now",
  id: "top-level-id",
  title: "Street Art Now",
  descriptionMarkdown:
    "A beach towel by [Yayoi Kusama](https://www.artsy.net/artist/yayoi-kusama), a classic print by [Alexander Calder](https://www.artsy.net/artist/alexander-calder), or a piggy bank by [Yoshitomo Nara](https://www.artsy.net/artist/yoshitomo-nara)â€”there are many ways to give the gift of art. Below, discover creative presents that your loved ones will treasure for years to come.",
  headerImage: "http://imageuploadedbymarketingteam.jpg",
  image: {
    id: "another-id",
    edges: [
      {
        node: { id: "an-id", image: { resized: { url: "https://defaultmostmarketableartworkincollectionimage.jpg" } } },
      },
    ],
  },
  collectionArtworks: {
    id: "QXJ0d29yazpjYXJvbHluLWJyYWR5LXN0b3VyaGVhZC1hY3Jvc3MtdGhlLWxha2U",
    pageInfo: {
      startCursor: "YXJyYXljb25uZWN0aW9uOjA=",
      endCursor: "YXJyYXljb25uZWN0aW9uOjc=",
      hasNextPage: false,
    },
    edges: [
      {
        cursor: "fds",
        id: "fdas",
        node: {
          __typename: "Artwork",
          id: "QXJ0d29yazpjYXJvbHluLWJyYWR5LXN0b3VyaGVhZC1hY3Jvc3MtdGhlLWxha2U=",
          slug: "carolyn-brady-stairheads-across-the-lake",
          image: {
            aspectRatio: 1.53,
            aspect_ratio: 1.53,
            url: "https://d32dm0rphc51dk.cloudfront.net/tH19ZfyCDocnNsKzXOZfHg/medium.jpg",
          },
          title: "Stourhead Across the Lake",
          date: "1997",
          sale_message: "$15,000",
          is_biddable: false,
          is_acquireable: false,
          is_offerable: false,
          sale: null,
          sale_artwork: null,
          artists: [
            {
              id: "carolyn-brady",
              name: "Carolyn Brady",
            },
          ],
          partner: {
            name: "Anderson Fine Art Gallery",
            id: "anderson-fine-art-gallery",
          },
          href: "/artwork/carolyn-brady-scoured-across-the-lake",
        },
      },
      {
        cursor: "fds",
        id: "fdas",
        node: {
          __typename: "Artwork",
          id: "QXJ0d29yazpjYXJvbHluLWJyYWR5LXN0b3VyaGVhZC1hY3Jvc3MtdGhlLWxha2U=",
          slug: "carolyn-brady-stairheads-across-the-lake",
          image: {
            aspectRatio: 1.53,
            aspect_ratio: 1.53,
            url: "https://d32dm0rphc51dk.cloudfront.net/tH19ZfyCDocnNsKzXOZfHg/medium.jpg",
          },
          title: "Stourhead Across the Lake",
          date: "1997",
          sale_message: "$15,000",
          is_biddable: false,
          is_acquireable: false,
          is_offerable: false,
          sale: null,
          sale_artwork: null,
          artists: [
            {
              id: "carolyn-brady",
              name: "Carolyn Brady",
            },
          ],
          partner: {
            name: "Anderson Fine Art Gallery",
            id: "anderson-fine-art-gallery",
          },
          href: "/artwork/carolyn-brady-scoured-across-the-lake",
        },
      },
      {
        cursor: "fds",
        id: "fdas",
        node: {
          __typename: "Artwork",
          id: "QXJ0d29yazpjYXJvbHluLWJyYWR5LXN0b3VyaGVhZC1hY3Jvc3MtdGhlLWxha2U=",
          slug: "carolyn-brady-stairheads-across-the-lake",
          image: {
            aspectRatio: 1.53,
            aspect_ratio: 1.53,
            url: "https://d32dm0rphc51dk.cloudfront.net/tH19ZfyCDocnNsKzXOZfHg/medium.jpg",
          },
          title: "Stourhead Across the Lake",
          date: "1997",
          sale_message: "$15,000",
          is_biddable: false,
          is_acquireable: false,
          is_offerable: false,
          sale: null,
          sale_artwork: null,
          artists: [
            {
              id: "carolyn-brady",
              name: "Carolyn Brady",
            },
          ],
          partner: {
            name: "Anderson Fine Art Gallery",
            id: "anderson-fine-art-gallery",
          },
          href: "/artwork/carolyn-brady-scoured-across-the-lake",
        },
      },
      {
        cursor: "fds",
        id: "fdas",
        node: {
          __typename: "Artwork",
          id: "QXJ0d29yazpjYXJvbHluLWJyYWR5LXN0b3VyaGVhZC1hY3Jvc3MtdGhlLWxha2U=",
          slug: "carolyn-brady-stairheads-across-the-lake",
          image: {
            aspectRatio: 1.53,
            aspect_ratio: 1.53,
            url: "https://d32dm0rphc51dk.cloudfront.net/tH19ZfyCDocnNsKzXOZfHg/medium.jpg",
          },
          title: "Stourhead Across the Lake",
          date: "1997",
          sale_message: "$15,000",
          is_biddable: false,
          is_acquireable: false,
          is_offerable: false,
          sale: null,
          sale_artwork: null,
          artists: [
            {
              id: "carolyn-brady",
              name: "Carolyn Brady",
            },
          ],
          partner: {
            name: "Anderson Fine Art Gallery",
            id: "anderson-fine-art-gallery",
          },
          href: "/artwork/carolyn-brady-scoured-across-the-lake",
        },
      },
      {
        cursor: "fds",
        id: "fdas",
        node: {
          __typename: "Artwork",
          id: "QXJ0d29yazpjYXJvbHluLWJyYWR5LXN0b3VyaGVhZC1hY3Jvc3MtdGhlLWxha2U=",
          slug: "carolyn-brady-stairheads-across-the-lake",
          image: {
            aspectRatio: 1.53,
            aspect_ratio: 1.53,
            url: "https://d32dm0rphc51dk.cloudfront.net/tH19ZfyCDocnNsKzXOZfHg/medium.jpg",
          },
          title: "Stourhead Across the Lake",
          date: "1997",
          sale_message: "$15,000",
          is_biddable: false,
          is_acquireable: false,
          is_offerable: false,
          sale: null,
          sale_artwork: null,
          artists: [
            {
              id: "carolyn-brady",
              name: "Carolyn Brady",
            },
          ],
          partner: {
            name: "Anderson Fine Art Gallery",
            id: "anderson-fine-art-gallery",
          },
          href: "/artwork/carolyn-brady-scoured-across-the-lake",
        },
      },
      {
        cursor: "fds",
        id: "fdas",
        node: {
          __typename: "Artwork",
          id: "QXJ0d29yazpjYXJvbHluLWJyYWR5LXN0b3VyaGVhZC1hY3Jvc3MtdGhlLWxha2U=",
          slug: "carolyn-brady-stairheads-across-the-lake",
          image: {
            aspectRatio: 1.53,
            aspect_ratio: 1.53,
            url: "https://d32dm0rphc51dk.cloudfront.net/tH19ZfyCDocnNsKzXOZfHg/medium.jpg",
          },
          title: "Stourhead Across the Lake",
          date: "1997",
          sale_message: "$15,000",
          is_biddable: false,
          is_acquireable: false,
          is_offerable: false,
          sale: null,
          sale_artwork: null,
          artists: [
            {
              id: "carolyn-brady",
              name: "Carolyn Brady",
            },
          ],
          partner: {
            name: "Anderson Fine Art Gallery",
            id: "anderson-fine-art-gallery",
          },
          href: "/artwork/carolyn-brady-scoured-across-the-lake",
        },
      },
    ],
  },
}
