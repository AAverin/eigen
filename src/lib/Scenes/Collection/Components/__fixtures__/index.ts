export const CollectionFixture = {
  collection: {
    title: "Street Art Now",
    description:
      '<p>From <a href="https://www.someartsyimage.net/artist/jean-michel-basquiat">Jean-Michel Basquiat</a> to <a href="https://www.someartsyimage.net/artist/shepard-fairey">Shepard Fairey</a>, street artists are known for their creative expressions of rebellion.</p>',
    headerImage: "http://imageuploadedbymarketingteam.jpg",
    image: { edges: [{ node: { imageUrl: "https://defaultmostmarketableartworkincollectionimage.jpg" } }] },
  },
}
