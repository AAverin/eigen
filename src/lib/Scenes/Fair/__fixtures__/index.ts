export const FairFixture = {
  id: "sofa-chicago-2018",
  name: "SOFA CHICAGO 2018",
  image: {
    image_url: "https://d32dm0rphc51dk.cloudfront.net/o0Zrtm-CgwbV1syD_33DuA/:version.jpg",
    aspect_ratio: 1,
    url: "https://d32dm0rphc51dk.cloudfront.net/o0Zrtm-CgwbV1syD_33DuA/square.jpg",
  },
  organizer: {
    profile: {
      icon: {
        id: "58ff95298b3b8104783c119c",
        href: null,
        height: 279,
        width: 279,
        url: "https://d32dm0rphc51dk.cloudfront.net/B1x-CqoWO1KitQnRj7PrrA/untouched-png.png",
      },
      name: "SOFA CHICAGO",
      __id: "UHJvZmlsZTpzb2ZhLWNoaWNhZ28=",
    },
  },
  start_at: "2018-11-01T17:00:00+00:00",
  end_at: "2018-11-04T18:00:00+00:00",
  location: {
    address: "",
    address_2: "",
    coordinates: {
      lat: 41.891551,
      lng: -87.6073751,
    },
    display: "41.891972, -87.605161",
    city: "Chicago",
    state: "Illinois",
    country: "United States",
    postal_code: "60611",
  },
  __id: "RmFpcjpzb2ZhLWNoaWNhZ28tMjAxOA==",
}
