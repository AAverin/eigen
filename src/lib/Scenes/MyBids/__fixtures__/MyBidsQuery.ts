export const sales = {
  edges: [
    {
      node: {
        saleType: "auction",
        href: "/auction/swann-auction-galleries-lgbtq-plus-art-material-culture-and-history",
        endAt: null,
        liveStartAt: "2020-08-13T16:00:00+00:00",
        displayTimelyAt: "live in 10d",
        timeZone: "America/New_York",
        name: "Swann Auction Galleries: LGBTQ+ Art, Material Culture & History",
        slug: "swann-auction-galleries-lgbtq-plus-art-material-culture-and-history",
        coverImage: {
          url: "https://d32dm0rphc51dk.cloudfront.net/1tJKlnBgHGvZSCZPlAvwAQ/wide.jpg",
        },
        partner: {
          name: "Swann Auction Galleries",
        },
      },
    },
    {
      node: {
        saleType: "auction",
        href: "/auction/heritage-urban-art-summer-skate",
        endAt: null,
        liveStartAt: "2020-08-05T15:00:00+00:00",
        displayTimelyAt: "live in 2d",
        timeZone: "America/Chicago",
        name: "Heritage: Urban Art Summer Skate",
        slug: "heritage-urban-art-summer-skate",
        coverImage: {
          url: "https://d32dm0rphc51dk.cloudfront.net/JOeiPjbfKixGJbQjQHubXA/source.jpg",
        },
        partner: {
          name: "Heritage Auctions",
        },
      },
    },
  ],
}

export const lotStandings = [
  {
    // Reserve not met
    isHighestBidder: false,
    isLeadingBidder: true,
    sale: {
      displayTimelyAt: "in progress",
      liveStartAt: "2020-08-03T19:10:10+00:00",
      endAt: null,
    },
    saleArtwork: {
      id: "U2FsZUFydHdvcms6NWYyODM2MzRmOTc5N2UwMDA3ZGM1MDU1",
      lotLabel: "3",
      counts: {
        bidderPositions: 1,
      },
      currentBid: {
        display: "CHF 1,800",
      },
      artwork: {
        artistNames: "Maskull Lasserre",
        href: "/artwork/maskull-lasserre-painting",
        image: {
          url: "https://d2v80f5yrouhh2.cloudfront.net/zrtyPc3hnFNl-1yv80qS2w/medium.jpg",
        },
      },
    },
  },
  {
    // Winning
    isHighestBidder: true,
    isLeadingBidder: true,
    sale: {
      displayTimelyAt: "in progress",
      liveStartAt: "2020-08-03T19:10:10+00:00",
      endAt: null,
    },
    saleArtwork: {
      id: "U2FsZUFydHdvcms6NWYyODM2MzNmOTc5N2UwMDA3ZGM1MDRk",
      lotLabel: "2",
      counts: {
        bidderPositions: 2,
      },
      currentBid: {
        display: "CHF 36,000",
      },
      artwork: {
        artistNames: "Zach Eugene Salinger-Simonson",
        href: "/artwork/zach-eugene-salinger-simonson-ennui",
        image: {
          url: "https://d2v80f5yrouhh2.cloudfront.net/NtVjXx1dzUIsOfFdyW0XZw/medium.jpg",
        },
      },
    },
  },
  {
    // Outbid
    isHighestBidder: false,
    isLeadingBidder: false,
    sale: {
      displayTimelyAt: "in progress",
      liveStartAt: "2020-08-03T19:10:10+00:00",
      endAt: null,
    },
    saleArtwork: {
      id: "U2FsZUFydHdvcms6NWYyODM2MzFmOTc5N2UwMDA3ZGM1MDQ1",
      lotLabel: "1",
      counts: {
        bidderPositions: 2,
      },
      currentBid: {
        display: "CHF 4,750",
      },
      artwork: {
        artistNames: "Leif Erik Nygards",
        href: "/artwork/leif-erik-nygards-surrealism",
        image: {
          url: "https://d2v80f5yrouhh2.cloudfront.net/dwqgENIWYbWFU_wntBtxFg/medium.jpg",
        },
      },
    },
  },
]
