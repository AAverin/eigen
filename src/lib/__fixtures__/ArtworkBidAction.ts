export const ArtworkFromAuctionPreview = {
  internalID: "artwork_from_preview_auction",
  sale: {
    registrationStatus: null,
    isPreview: true,
    isOpen: false,
    isLiveOpen: false,
    isClosed: false,
    isRegistrationClosed: false,
  },
  saleArtwork: {
    increments: [
      { cents: 400000, display: "CHF4,000" },
      { cents: 425000, display: "CHF4,250" },
      { cents: 450000, display: "CHF4,500" },
      { cents: 475000, display: "CHF4,750" },
    ],
  },
  myLotStanding: null,
}

export const ArtworkFromTimedAuctionRegistrationOpen = {
  internalID: "artwork_from_open_non_live_auction",
  sale: {
    registrationStatus: null,
    isPreview: false,
    isOpen: true,
    isLiveOpen: false,
    isClosed: false,
    isRegistrationClosed: false,
  },
  saleArtwork: {
    increments: [
      { cents: 90000, display: "$900" },
      { cents: 95000, display: "$950" },
      { cents: 100000, display: "$1,000" },
      { cents: 110000, display: "$1,100" },
    ],
  },
  myLotStanding: null,
}

export const ArtworkFromTimedAuctionRegistrationClosed = {
  internalID: "artwork_from_open_non_live_auction",
  sale: {
    registrationStatus: null,
    isPreview: false,
    isOpen: true,
    isLiveOpen: false,
    isClosed: false,
    isRegistrationClosed: true,
  },
  saleArtwork: {
    increments: [
      { cents: 750000, display: "$7,500" },
      { cents: 800000, display: "$8,000" },
      { cents: 850000, display: "$8,500" },
      { cents: 900000, display: "$9,000" },
    ],
  },
  myLotStanding: null,
}

export const ArtworkFromLiveAuctionRegistrationOpen = {
  internalID: "artwork_from_open_live_auction_open_registration",
  sale: {
    registrationStatus: null,
    isPreview: false,
    isOpen: true,
    isLiveOpen: true,
    isClosed: false,
    isRegistrationClosed: false,
  },
  saleArtwork: {
    increments: [
      { cents: 320000, display: "€3,200" },
      { cents: 350000, display: "€3,500" },
      { cents: 380000, display: "€3,800" },
      { cents: 400000, display: "€4,000" },
    ],
  },
  myLotStanding: null,
}

export const ArtworkFromLiveAuctionRegistrationClosed = {
  internalID: "artwork_from_open_live_auction_closed_registration",
  sale: {
    registrationStatus: null,
    isPreview: false,
    isOpen: true,
    isLiveOpen: true,
    isClosed: false,
    isRegistrationClosed: true,
  },
  saleArtwork: {
    increments: [
      { cents: 320000, display: "€3,200" },
      { cents: 350000, display: "€3,500" },
      { cents: 380000, display: "€3,800" },
      { cents: 400000, display: "€4,000" },
    ],
  },
  myLotStanding: null,
}

export const ArtworkFromClosedAuction = {
  internalID: "artwork_from_closed_auction",
  sale: {
    registrationStatus: null,
    isPreview: false,
    isOpen: false,
    isLiveOpen: false,
    isClosed: true,
    isRegistrationClosed: false,
  },
  saleArtwork: {
    increments: [
      { cents: 425000, display: "£4,250" },
      { cents: 450000, display: "£4,500" },
      { cents: 475000, display: "£4,750" },
      { cents: 500000, display: "£5,000" },
    ],
  },
  myLotStanding: null,
}

export const NotRegisteredToBid = {
  myLotStanding: null,
  sale: {
    registrationStatus: null,
  },
}

export const BidderPendingApproval = {
  myLotStanding: null,
  sale: {
    registrationStatus: {
      id: "bidder_pending_approval",
      qualifiedForBidding: false,
    },
  },
}

export const RegisteredBidder = {
  myLotStanding: null,
  sale: {
    registrationStatus: { id: "bidder_approved", qualifiedForBidding: true },
  },
}

export const RegistedBidderWithBids = {
  myLotStanding: [
    {
      mostRecentBid: {
        maxBid: {
          cents: 30000,
        },
      },
    },
  ],
  sale: {
    registrationStatus: { id: "bidder_approved", qualifiedForBidding: true },
  },
}
