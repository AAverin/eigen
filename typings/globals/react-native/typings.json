{
  "resolution": "main",
  "tree": {
    "src": "https://raw.githubusercontent.com/DefinitelyTyped/DefinitelyTyped/4c7d03b088d4e71cd9fe9c66ab7bd363049077c3/react-native/react-native.d.ts",
    "raw": "registry:dt/react-native#0.29.0+20160719232534",
    "typings": "https://raw.githubusercontent.com/DefinitelyTyped/DefinitelyTyped/4c7d03b088d4e71cd9fe9c66ab7bd363049077c3/react-native/react-native.d.ts"
  }
}
