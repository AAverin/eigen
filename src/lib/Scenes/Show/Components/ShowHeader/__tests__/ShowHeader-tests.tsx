import { ShowFixture } from "lib/__fixtures__/ShowFixture"
import React from "react"
import "react-native"
import * as renderer from "react-test-renderer"
import { ShowHeader } from "../index"
it("looks correct when rendered", () => {
  const onSaveShowPressed = () => Promise.resolve()
  const onViewAllArtistsPressed = () => {
    /** noop */
  }
  const comp = renderer.create(
    <ShowHeader
      show={ShowFixture as any}
      onSaveShowPressed={onSaveShowPressed}
      onViewAllArtistsPressed={onViewAllArtistsPressed}
    />
  )
  expect(comp).toMatchSnapshot()
})
