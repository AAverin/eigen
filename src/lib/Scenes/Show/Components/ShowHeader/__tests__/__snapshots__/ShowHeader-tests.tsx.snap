// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`looks correct when rendered 1`] = `
Array [
  <View
    pb={0}
    pt={3}
    px={2}
    style={
      Array [
        Object {
          "paddingBottom": 0,
          "paddingLeft": 8,
          "paddingRight": 8,
          "paddingTop": 16,
        },
        undefined,
      ]
    }
  >
    <View
      m={2}
      style={
        Array [
          Object {
            "marginBottom": 8,
            "marginLeft": 8,
            "marginRight": 8,
            "marginTop": 8,
          },
          undefined,
        ]
      }
    />
    <Text
      accessible={true}
      allowFontScaling={true}
      className="sc-bdVaJa bDWFJH"
      ellipsizeMode="tail"
      fontFamily="Unica77LL-Regular"
      fontSize="14px"
      lineHeight="24px"
      mb={0.5}
      style={
        Array [
          Object {
            "fontFamily": "Unica77LL-Regular",
            "fontSize": 14,
            "lineHeight": 24,
            "marginBottom": 0.5,
          },
          undefined,
        ]
      }
    >
      Anderson Fine Art Gallery
    </Text>
    <Text
      accessible={true}
      allowFontScaling={true}
      className="sc-bwzfXH uTxCW"
      ellipsizeMode="tail"
      fontFamily="AGaramondPro-Regular"
      fontSize="32px"
      lineHeight={34}
      style={
        Array [
          Object {
            "fontFamily": "AGaramondPro-Regular",
            "fontSize": 32,
            "lineHeight": 34,
          },
          undefined,
        ]
      }
    >
      Flickinger Collection
    </Text>
    <Text
      accessible={true}
      allowFontScaling={true}
      className="sc-bdVaJa bDWFJH"
      ellipsizeMode="tail"
      fontFamily="Unica77LL-Regular"
      fontSize="14px"
      lineHeight="24px"
      style={
        Array [
          Object {
            "fontFamily": "Unica77LL-Regular",
            "fontSize": 14,
            "lineHeight": 24,
          },
          undefined,
        ]
      }
    >
      Jul 1 â€“ Oct 30
    </Text>
  </View>,
  <View
    my={2}
    style={
      Array [
        Object {
          "marginBottom": 8,
          "marginTop": 8,
        },
        undefined,
      ]
    }
  >
    <RCTScrollView
      horizontal={true}
      scrollEventThrottle={160}
      showsHorizontalScrollIndicator={false}
      style={
        Array [
          Object {
            "height": 350,
            "paddingLeft": 20,
          },
          undefined,
        ]
      }
    >
      <View>
        <View
          alignItems="flex-start"
          mr={1}
          style={
            Array [
              Object {
                "alignItems": "flex-start",
                "display": "flex",
                "marginRight": 4,
              },
              undefined,
            ]
          }
        >
          <AROpaqueImageView
            aspectRatio={0.87}
            imageURL="https://d32dm0rphc51dk.cloudfront.net/Bu7ONuoTlR-PLZDUjs6ZTA/medium.jpg"
            isFirst={true}
            style={
              Array [
                Object {
                  "height": 200,
                  "marginTop": 150,
                  "width": 650,
                },
                undefined,
              ]
            }
          />
        </View>
      </View>
    </RCTScrollView>
  </View>,
  <View
    mb={4}
    px={2}
    style={
      Array [
        Object {
          "marginBottom": 32,
          "paddingLeft": 8,
          "paddingRight": 8,
        },
        undefined,
      ]
    }
  >
    <View
      flexDirection="row"
      flexWrap="wrap"
      mb="8"
      style={
        Array [
          Object {
            "display": "flex",
            "flexDirection": "row",
            "flexWrap": "wrap",
            "marginBottom": 512,
          },
          undefined,
        ]
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        className="sc-bdVaJa bDWFJH"
        ellipsizeMode="tail"
        fontFamily="Unica77LL-Regular"
        fontSize="14px"
        lineHeight="19"
        style={
          Array [
            Object {
              "fontFamily": "Unica77LL-Regular",
              "fontSize": 14,
              "lineHeight": 19,
            },
            undefined,
          ]
        }
      >
        Works by 
      </Text>
      <View
        accessible={true}
        isTVSelectable={true}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "opacity": 1,
          }
        }
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          className="sc-bdVaJa bDWFJH"
          ellipsizeMode="tail"
          fontFamily="Unica77LL-Medium"
          fontSize="14px"
          lineHeight="19"
          style={
            Array [
              Object {
                "fontFamily": "Unica77LL-Medium",
                "fontSize": 14,
                "lineHeight": 19,
              },
              undefined,
            ]
          }
        >
          Hans Hofmann, 
        </Text>
      </View>
      <View
        accessible={true}
        isTVSelectable={true}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "opacity": 1,
          }
        }
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          className="sc-bdVaJa bDWFJH"
          ellipsizeMode="tail"
          fontFamily="Unica77LL-Medium"
          fontSize="14px"
          lineHeight="19"
          style={
            Array [
              Object {
                "fontFamily": "Unica77LL-Medium",
                "fontSize": 14,
                "lineHeight": 19,
              },
              undefined,
            ]
          }
        >
          David Hockney
        </Text>
      </View>
      <Text
        accessible={true}
        allowFontScaling={true}
        className="sc-bdVaJa bDWFJH"
        ellipsizeMode="tail"
        fontFamily="Unica77LL-Regular"
        fontSize="14px"
        lineHeight="19"
        style={
          Array [
            Object {
              "fontFamily": "Unica77LL-Regular",
              "fontSize": 14,
              "lineHeight": 19,
            },
            undefined,
          ]
        }
      >
         and 
      </Text>
      <View
        accessible={true}
        isTVSelectable={true}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "opacity": 1,
          }
        }
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          className="sc-bdVaJa bDWFJH"
          ellipsizeMode="tail"
          fontFamily="Unica77LL-Medium"
          fontSize="14px"
          lineHeight="19"
          style={
            Array [
              Object {
                "fontFamily": "Unica77LL-Medium",
                "fontSize": 14,
                "lineHeight": 19,
              },
              undefined,
            ]
          }
        >
          16 others
        </Text>
      </View>
    </View>
    <View
      mt={1}
      style={
        Array [
          Object {
            "marginTop": 4,
          },
          undefined,
        ]
      }
    />
    <View
      accessible={true}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "rgba(0, 0, 0, 1)",
          "borderColor": "rgba(0, 0, 0, 1)",
          "borderRadius": 2,
          "borderWidth": 1,
          "height": 46,
          "justifyContent": "center",
          "opacity": 1,
        }
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        className="sc-bdVaJa bDWFJH"
        color="white"
        ellipsizeMode="tail"
        fontFamily="Unica77LL-Medium"
        fontSize="14px"
        lineHeight="24px"
        style={
          Array [
            Object {
              "color": "white",
              "fontFamily": "Unica77LL-Medium",
              "fontSize": 14,
              "lineHeight": 24,
            },
            undefined,
          ]
        }
      >
        Save show
      </Text>
    </View>
    <View
      m={1}
      style={
        Array [
          Object {
            "marginBottom": 4,
            "marginLeft": 4,
            "marginRight": 4,
            "marginTop": 4,
          },
          undefined,
        ]
      }
    />
    <Text
      accessible={true}
      allowFontScaling={true}
      className="sc-bwzfXH uTxCW"
      ellipsizeMode="tail"
      fontFamily="AGaramondPro-Regular"
      fontSize="16px"
      lineHeight="20px"
      style={
        Array [
          Object {
            "fontFamily": "AGaramondPro-Regular",
            "fontSize": 16,
            "lineHeight": 20,
          },
          undefined,
        ]
      }
    >
      Masterworks on paper, prints, paintings, and sculptures by mainstay postwar and contemporary artists are the focus in this exclusive exhibition showcasing the discerning connoisseurship of Tom & Penny Flickinger, lifelong collectors, and trustees of the Albright Knox Art Gallery.
    </Text>
    <View
      m={1}
      style={
        Array [
          Object {
            "marginBottom": 4,
            "marginLeft": 4,
            "marginRight": 4,
            "marginTop": 4,
          },
          undefined,
        ]
      }
    />
    <View
      accessible={true}
      align-items="base-line"
      flexDirection="row"
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Array [
          Object {
            "display": "flex",
            "flexDirection": "row",
          },
          undefined,
        ]
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        className="sc-bdVaJa bDWFJH"
        ellipsizeMode="tail"
        fontFamily="Unica77LL-Medium"
        fontSize="14px"
        lineHeight="20px"
        style={
          Array [
            Object {
              "fontFamily": "Unica77LL-Medium",
              "fontSize": 14,
              "lineHeight": 20,
            },
            undefined,
          ]
        }
      >
        View more information
      </Text>
      <View
        ml={1}
        style={
          Array [
            Object {
              "marginLeft": 4,
            },
            undefined,
          ]
        }
      >
        <RNSVGSvgView
          align="xMidYMid"
          bbHeight={16}
          bbWidth={16}
          height={16}
          meetOrSlice={0}
          minX={0}
          minY={0}
          style={
            Array [
              Object {
                "backgroundColor": "transparent",
                "borderWidth": 0,
              },
              Object {
                "top": 1,
              },
              false,
              Object {
                "flex": 0,
                "height": 16,
                "width": 16,
              },
            ]
          }
          vbHeight={54}
          vbWidth={54}
          width={16}
        >
          <RNSVGGroup
            fill={
              Array [
                0,
                0,
                0,
                0,
                1,
              ]
            }
            fillOpacity={1}
            fillRule={1}
            font={Object {}}
            matrix={
              Array [
                1,
                0,
                -0,
                1,
                0,
                0,
              ]
            }
            opacity={1}
            originX={0}
            originY={0}
            propList={
              Array [
                "fill",
                "fillOpacity",
                "fillRule",
                "stroke",
                "strokeWidth",
                "strokeOpacity",
                "strokeDasharray",
                "strokeDashoffset",
                "strokeLinecap",
                "strokeLinejoin",
                "strokeMiterlimit",
              ]
            }
            rotation={0}
            scaleX={1}
            scaleY={1}
            skewX={0}
            skewY={0}
            stroke={null}
            strokeDasharray={null}
            strokeDashoffset={null}
            strokeLinecap={0}
            strokeLinejoin={0}
            strokeMiterlimit={4}
            strokeOpacity={1}
            strokeWidth={1}
            x={0}
            y={0}
          >
            <RNSVGGroup
              fill={
                Array [
                  0,
                  0,
                  0,
                  0,
                  1,
                ]
              }
              fillOpacity={1}
              fillRule={1}
              font={Object {}}
              matrix={
                Array [
                  1,
                  0,
                  -0,
                  1,
                  0,
                  0,
                ]
              }
              opacity={1}
              originX={0}
              originY={0}
              propList={Array []}
              rotation={0}
              scaleX={1}
              scaleY={1}
              skewX={0}
              skewY={0}
              stroke={null}
              strokeDasharray={null}
              strokeDashoffset={null}
              strokeLinecap={0}
              strokeLinejoin={0}
              strokeMiterlimit={4}
              strokeOpacity={1}
              strokeWidth={1}
              x={0}
              y={0}
            >
              <RNSVGPath
                d="M18.192 7.92L32.498 27 18.192 46.08a1 1 0 0 0 .2 1.4l1.6 1.2a1 1 0 0 0 1.4-.2L36.597 28.2a2 2 0 0 0 0-2.4L21.392 5.52a1 1 0 0 0-1.4-.2l-1.6 1.2a1 1 0 0 0-.2 1.4z"
                fill={
                  Array [
                    0,
                    0.7607843137254902,
                    0.7607843137254902,
                    0.7607843137254902,
                    1,
                  ]
                }
                fillOpacity={1}
                fillRule={0}
                matrix={
                  Array [
                    1,
                    0,
                    -0,
                    1,
                    0,
                    0,
                  ]
                }
                opacity={1}
                originX={0}
                originY={0}
                propList={
                  Array [
                    "fill",
                    "fillRule",
                  ]
                }
                rotation={0}
                scaleX={1}
                scaleY={1}
                skewX={0}
                skewY={0}
                stroke={null}
                strokeDasharray={null}
                strokeDashoffset={null}
                strokeLinecap={0}
                strokeLinejoin={0}
                strokeMiterlimit={4}
                strokeOpacity={1}
                strokeWidth={1}
                x={0}
                y={0}
              />
            </RNSVGGroup>
          </RNSVGGroup>
        </RNSVGSvgView>
      </View>
    </View>
  </View>,
]
`;
